sos/report/plugins/leapp.py

```diff
@@ -16,10 +16,13 @@ class Leapp(Plugin, RedHatPlugin):
     short_desc = 'Leapp upgrade handling tool'
 
     plugin_name = 'leapp'
-    packages = ('leapp', 'leapp-repository')
+    packages = ('leapp',)
```

Comment:
In which version of RHEL did we stop distributing leapp-repository? 
Or in other words - will this stop the plugin executing in old RHELs where migration could potentially still happen?

Reply:
https://github.com/oamg/leapp-repository/commit/924c398a003f8ada079f2819fa3305adda7dfcda is in leapp-repository (SRPM name!) 0.15+, which is RHEL 7.9+
Pretty sure older RHEL 7.x is not supported anymore for leapp, and even if: the `leapp` package is present in any case (unless the user uninstalled it, ofc).

Reply:
Perfect, thank you for clarifying this!